src/dmd/astbase.d

```diff
@@ -1483,12 +1483,19 @@ struct ASTBase
 
         const FileName srcfile;
         const(char)* arg;
+        bool rootModule;
```

Comment:
should be a flag in a bitfield not a standalone bool.
It's a good time to start doing this.